src/Codec/FFmpeg/Decode.hs

```diff
@@ -9,8 +9,8 @@ import Codec.FFmpeg.Scaler
 import Codec.FFmpeg.Types
 import Control.Arrow (first)
 import Control.Monad (when, void)
-import Control.Monad.Except
-import Control.Monad.Trans.Maybe
+import Control.Monad.Except (MonadIO, MonadError, liftIO, throwError, catchError)
```

Comment:
For ghc-8.8.1, without using explicit imports here, somehow `when` and `void` get imported here, leading to a warning about unnecessary imports for `Control.Monad (when, void)`.